manifests/init.pp

```diff
@@ -29,8 +29,10 @@
   $tomcat = $gutterball::params::tomcat,
 ) inherits gutterball::params {
 
+  $_ = cache_data('gutterball_db_password', $dbpassword)
+  $__ = cache_data('gutterball_db_user', $dbuser)
+  $___ = cache_data('gutterball_tomcat', $tomcat)
```

Comment:
I agree with @ehelms on this. Also, a short comment that it's needed for the installer may be useful to prevent others from the same confusion as I had.